src/all/apt/String.cpp

```diff
@@ -317,27 +317,33 @@ StringBase::StringBase(StringBase&& _rhs_)
 	, m_capacity(_rhs_.m_capacity)
 	, m_length(_rhs_.m_length)
 {
-	if (_rhs_.isLocal()) {
+	if (_rhs_.isLocal())
 		strncpy(m_buf, _rhs_.m_buf, _rhs_.m_length + 1);
-	} else {
+	else
 		m_buf = _rhs_.m_buf;
-		_rhs_.m_buf = nullptr;
-		_rhs_.m_capacity = 0;
-		_rhs_.m_length = 0;
-	}
+	m_capacity = _rhs_.m_capacity;
+	m_length = _rhs_.m_length;
```

Comment:
`m_capacity` and `m_length` are already set by the member initializer list (lines 317-318 above)